pkg/sql/plan2/function/builtin/multi/pi.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 - 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multi
+
+import (
+	"errors"
+	"github.com/matrixorigin/matrixone/pkg/container/nulls"
+	"github.com/matrixorigin/matrixone/pkg/container/types"
+	"github.com/matrixorigin/matrixone/pkg/container/vector"
+	"github.com/matrixorigin/matrixone/pkg/vectorize/pi"
+	"github.com/matrixorigin/matrixone/pkg/vm/process"
+)
+
+func Pi(lv []*vector.Vector, proc *process.Process) (*vector.Vector, error) {
+	if len(lv) != 0 {
```

Comment:
redundant check

Reply:
You're right. It's a redundancy check. Let me correct it